File: C-like lessons/C++ lessons/UnitTest/TestsFile.cpp
#include "pch.h"
#include "CppUnitTest.h"
#include "..\Lesson\Lesson.cpp"

using namespace Microsoft::VisualStudio::CppUnitTestFramework;

namespace UnitTest
{
	TEST_CLASS(UnitTest)
	{
	public:
		
		TEST_METHOD(CallingWithOneArgument)
		{
			Assert::AreEqual(Increment(2), 3);
		}
		TEST_METHOD(CallingWithTwoArguments)
		{
			Assert::AreEqual(Increment(2, 2), 4);
		}
	};
}

File: C-like lessons/CS lessons/Test/TestsFile.cs
using System;
using Lessons;
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace Tests
{
    [TestClass]
    public class TestsFile
    {
        bool Expected = true;
        int[] Input = {368, 276, 460};
        [TestMethod]
        public void Test()
        {
            Assert.AreEqual(Expected, Methods.IsRightTriangle(Input));
        }
    }
}
